src/wallet3/cli-wallet/oxen_wallet_cli/walletcli.py

```diff
@@ -98,12 +112,67 @@ def address():
     click.echo("Address: {}".format("TODO sean get the address here"))
 
 @walletcli.command()
-def get_balance():
-    click.echo("Balance: {}".format(context.wallet.get_balance()))
+def balance():
+    if context.wallet is None:
+        click.echo("Wallet not loaded")
+        return
+    click.echo("Balance: {:.2f} Oxen".format(context.wallet.get_balance()/1e9))
```

Comment:
This rounding needs to be configurable (if/until it isn't, it should just go full precision).

Reply:
https://stackoverflow.com/questions/41853132/how-to-set-variable-precision-for-new-python-format-option

Looks possible!